src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -237,11 +236,24 @@ def check_each_element_for_error(instance):
             )
 
 
+def check_only_one_of_each_element(instance):
+    if "elements" not in instance:
+        return
+    found = {}
+    for element in instance["elements"]:
+        if element["type"]:
+            found[element["type"]] = found.get(element["type"], 0) + 1
+
+            if found[element["type"]] > 1:
+                raise SchemaValidationError(f"Multiple elements of type: {element['type']}")
```

Comment:
you could make this a little simpler by checking for a dupe before adding to the dict.
```suggestion
    for element in instance["elements"]:
        if element["type"]:
            if element["type"] not in found:
                found[element["type"]] = 1
            else:
                raise SchemaValidationError(f"Multiple elements of type: {element['type']}")
```